prestoadmin/standalone/config.py

```diff
@@ -140,6 +143,24 @@ def validate_workers(workers):
     return workers
 
 
+def expand_host(host):
+    m = re.match("(.*)\[(\d{1,})-(\d{1,})\](.*)", host)
+    if m is not None and len(m.groups()) == 4:
```

Comment:
I think it would be nice to model it in more object-oriented manner. So it it would be possible to inject different expansions strategies.